launcher/ui/pages/modplatform/ftb/FtbPage.cpp

```diff
@@ -126,7 +127,7 @@ void FtbPage::suggestCurrent()
         return;
     }
 
-    dialog->setSuggestedPack(selected.name + " " + selectedVersion, new ModpacksCH::PackInstallTask(selected, selectedVersion));
+    dialog->setSuggestedPack(selected.name + " " + selectedVersion, new ModpacksCH::PackInstallTask(selected, selectedVersion, this));
```

Comment:
this seems wrong?

Reply:
Sort of. Previously (when there was a seperation of api and application), you wouldn't have been able to do this.

It's why there is a `UserInteractionSupport` class for the ATLauncher support.